### ISSUE TITLE
[Bug]: V Rising

### ISSUE THREAD

**Issue by Shankoh** [parkervcp/eggs#2265]
### Panel Version

1.11.3

### Wings Version

1.11.6

### Service

v_rising /v_rising_vanilla

### Modified

No, I did not modify the egg

### Expected Behavior

Getting this spammed

### Actual Behavior

SteamLog [SDR k_ESteamNetworkingSocketsDebugOutputType_Warning] SteamNetworkingS
ockets lock held for 10.8ms.  (Performance warning.)  SendMessageToConnection
This is usually a symptom of a general performance problem such as thread starva
tion.
SteamLog [SDR k_ESteamNetworkingSocketsDebugOutputType_Warning] SteamNetworkingS
ockets lock held for 5.2ms.  (Performance warning.)  ServiceThread,RecvUDPPacket
(x2),ThinkPingProbes,P2P ThinkSelectServer,ThinkSelectServer,EnsureDataCenterRou
tesValid
This is usually a symptom of a general performance problem such as thread starva
tion.

### Steps To Reproduce

Start V Rising
Console spam 

### Install logs

https://pteropaste.com/gdn38kzsi70q/

**Comment by lilkingjr1** [comment]
Did you read the error? Can you please prove you're not suffering from thread starvation?

**Comment by Shankoh** [comment]
CPU Limit | 500%
CPU Pinning | 8,9,10,11,12
Memory | Unlimited / Not Set (16 gb)

**Comment by lilkingjr1** [comment]
Have you tried without CPU pinning? I'm not sure how well Wine works with that.

**Comment by Shankoh** [comment]
Yeah I have, still get the same message not too sure why. Is there a config that might be overwriting it?

**Comment by danny6167** [comment]
We see this error as well, however users report that the servers are performing great.
We only seem to get it every minute or so.

Plenty of free memory and CPU time. 200% CPU limit. Haven't messed with thread limits because we haven't seen any issues from it.

**Comment by lilkingjr1** [comment]
Closing as wontfix for now, as console spam from the game isn't generally considered an Egg bug, especially if the server runs fine and the messages pertain to performance or Steam connections (see Minecraft, Arma, etc). We can reopen the issue if you can provide more evidence that it has a significant impact on the server.

**Comment by joaquinv98** [comment]
Same issue here, when i get that error the game does not appear in the server list. I'm 99% sure that it can't be a hardware issue, 5900x should be enough

**Comment by vanderlpp** [comment]
still getting the same message. any resolution of this problem?
In my opinion, it seems to be a problem connecting to Steam.

**Comment by lilkingjr1** [comment]
> still getting the same message. any resolution of this problem? In my opinion, it seems to be a problem connecting to Steam.
> 
> The problem causes disconnection every 5 minutes. the more people playing, the more disconnections there will be. I even changed the save time, which was 5 minutes, but it didn't solve the problem. So the cause would be something with Steam. but it is disconnecting players and taking them backwards when walking around the game environment.

@vanderlpp Good observation. Would it be possible for you to see if this issue persists in a more "vanilla" Linux or (even better) Windows environment to rule out a general Steam or Game Server related issue out of our control? In other words, I recognize there may still be an issue, but I am unsure on how to troubleshoot it easily. If we can confirm the issue lies within Pterodactyl, I will re-open this issue.

On a personal note: I get this message from time to time on my small personal server, but not reliably every 5 min. However, only 3 people play and I connect directly, so I don't notice any negatives at all. Also, it lends credence to it possibly being related to strain via more player connections as you stated. It's just hard to say if this is a natural V-Rising/Steam issue or something related more to Pterodactyl.

**Comment by vanderlpp** [comment]
> > ainda recebendo a mesma mensagem. alguma resolução deste problema? Na minha opinião, parece ser um problema de conexão com o Steam.
> > O problema causa desconexão a cada 5 minutos. quanto mais gente jogando, mais desconexões haverá. Até mudei o tempo de salvamento, que era de 5 minutos, mas não resolveu o problema. Então a causa seria algo com o Steam. mas está desconectando os jogadores e levando-os para trás ao caminhar pelo ambiente do jogo.
> 
> @vanderlppBoa observação. Seria possível para você ver se esse problema persiste em um ambiente Linux mais "vanilla" ou (melhor ainda) Windows para descartar um problema geral relacionado ao Steam ou ao servidor de jogos fora de nosso controle? Em outras palavras, reconheço que ainda pode haver um problema, mas não tenho certeza de como solucioná-lo facilmente. Se pudermos confirmar que o problema está no Pterodáctilo, reabrirei este problema.
> 
> A título pessoal: recebo esta mensagem de vez em quando no meu pequeno servidor pessoal, mas não de forma confiável a cada 5 minutos. No entanto, apenas 3 pessoas jogam e eu me conecto diretamente, então não percebo nenhum ponto negativo. Além disso, dá credibilidade ao fato de estar possivelmente relacionado à tensão por meio de mais conexões de jogadores, como você afirmou. É difícil dizer se este é um problema natural do V-Rising/Steam ou algo mais relacionado ao Pterodáctilo.

It still happens if the game runs on Windows.

Vrising Server logo on Windows, since installation.

I used the windowsGSM program to run the game because it was easier and because I already had it installed.

[LOG](https://pastebin.com/r1p8Q28j)
